project/Settings.scala

```diff
@@ -0,0 +1,56 @@
+import Dependencies._
+import Merging._
+import Testing._
+import Version._
+import sbt.Keys._
+import sbt._
+import sbtassembly.AssemblyPlugin.autoImport._
+
+object Settings {
+
+  val artifactory = "https://artifactory.broadinstitute.org/artifactory/"
+
+  val commonResolvers = List(
+    "artifactory-releases" at artifactory + "libs-release",
+    "artifactory-snapshots" at artifactory + "libs-snapshot"
+  )
+
+  //coreDefaultSettings + defaultConfigs = the now deprecated defaultSettings
+  val commonBuildSettings = Defaults.coreDefaultSettings ++ Defaults.defaultConfigs ++ Seq(
+    javaOptions += "-Xmx2G",
+    javacOptions ++= Seq("-source", "1.8", "-target", "1.8")
+  )
+
+  val commonCompilerSettings = Seq(
+    "-unchecked",
+    "-deprecation",
+    "-feature",
+    "-encoding", "utf8",
```

Comment:
Maybe it could? I just kept what we had before: https://github.com/broadinstitute/firecloud-orchestration/pull/372/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceL45